data/coaches.yml

```diff
@@ -0,0 +1,115 @@
+-
+  name: Silvana Precup
+  description: Secretary
+  image: silvana.jpg
+  twitter: silvanaprecup
+-
+  name: Oana Şipoş
+  description:
+  image:
+  twitter: oanasipos
+-
+  name: Bert Goethals
+  description:
+  image:
+  twitter: bertgoethals
+-
+  name: Anne Collet
+  description:
+  image:
+  twitter: DianeLanterman
+-
+  name: Christophe Philemotte
+  description: President
+  image: toch.jpeg
+  twitter: _toch
+-
+  name: Martin Van Aken
+  description:
+  image:
+  twitter: martinvanaken
+-
+  name: Hannes Fostie
+  description:
+  image:
+  twitter: hannesfostie
+-
+  name: Yannick Schutz
+  description:
+  image:
+  twitter: yann_ck
+-
+  name: Daniel Reszka
+  description:
+  image:
+  twitter: pixarea
+-
+  name: Nathan Van der Auwera
+  description:
+  image:
+  twitter: dixis
+-
+  name: Joren De Groof
+  description: Treasurer
+  image: joren.jpeg
+  twitter: joren
+-
+  name: Dimitri De Frenne
+  description:
+  image:
+  twitter: ddefrenne
+-
+  name: Kenneth Geerts
+  description:
+  image:
+  twitter: kennethgeerts
+-
+  name: Marie du Chastel
+  description:
+  image:
+  twitter: mariekik
+-
+  name: Marius Andra
+  description:
+  image:
+  twitter: mariusandra
+-
+  name: Arne Brasseur
+  description:
+  image:
+  twitter: plexus
+-
+  name: Loic Vigneron
+  description:
+  image:
+  twitter: loicvigneron
+-
+  name: Ayrton De Craene
+  description:
+  image:
+  twitter: ayrtonbe
+-
+  name: Kristof Overdulve
+  description:
+  image:
+  twitter: KristofOverdulve
+-
+  name: Raphael Doehring
+  description:
+  image:
+  twitter: wrtsprt
+-
+  name: Benjamin Roth
+  description:
+  image:
+  twitter: apneadiving
+-
+  name: Javier Sanz
+  description:
+  image:
+  twitter: malditojavi
+-
+  name: Laura Gaetano
+  description:
+  image:
+  twitter: alicetragedy
```

Comment:
You forgot about me ;) Also are old coaches, not sure about their presence for 2016. Maybe we shouldnt mention the date at all ^^